tests/skydb_integration_test.go

```diff
@@ -0,0 +1,278 @@
+package tests
```

Comment:
Can you please also add tests against the registry? We have some in `skynet-js` you can look at.

Reply:
Also I think we should have at least one end-to-end test that sets a random registry entry on siasky.net, queries the entry, and makes sure we get the same data back.